Update alexa.intent.markdown (#21308)
home-assistant/home-assistant.github.io: source/_integrations/alexa.intent.markdown

  - DEBUG *(optional)*: set to *True* to log debugging messages.
  - LONG_LIVED_ACCESS_TOKEN *(optional, not recommended)*: you will connect your Alexa Custom skill with your Home Assistant user account in the later steps, so that you don't need to use long-lived access token here. However, the access token you got from login flow is only valid for 30 minutes. It will be hard for you to test lambda function with the access token in test data. So for your convenience, you can remove the access token from the test data, [generate a long-lived access token][generate-long-lived-access-token] to put here, then the function will fall back to reading the token from environment variables. (tips: You did not enable the security storage for your environment variables, so your token saved here is not that safe. You should only use it for debugging and testing purpose. You should remove and delete the long-lived access token after you finish the debugging.)
- Save your environmental variables by clicking the `Save` button.

> - Next, copy the ARN displayed in the top of the page, which is the identify of this Lambda function. 

Reviewer: ```suggestion
- Next, copy the ARN displayed at the top of the page, which is the identity of this Lambda function. 
```

>   - Go back to your Alexa skill and go to Custom->Endpoint menu option on the left.

Reviewer: ```suggestion
  - Go back to your Alexa skill and go to the Custom->Endpoint menu option on the left.
```

>   - Paste the ARN value in the "Default Region".  Note: you will not be able to do this until you have completed the step above adding the Alexa Skills Kit trigger (done in previous step) to the AWS Lambda Function.

Reviewer: ```suggestion
  - Paste the ARN value in the "Default Region". Note: you will not be able to do this until you have completed the step above adding the Alexa Skills Kit trigger (done in the previous step) to the AWS Lambda Function.
```
